src/smoothscroll.js

```diff
@@ -158,6 +158,10 @@ function polyfill() {
   function findScrollableParent(el) {
     var isBody;
 
+    if (el === d.body) {
```

Comment:
if we did `var isBody = el === d.body` would this work? Instead of early exiting the rest of the logic will continue working as expected I think.